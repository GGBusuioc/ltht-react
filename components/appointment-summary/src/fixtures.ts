import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
