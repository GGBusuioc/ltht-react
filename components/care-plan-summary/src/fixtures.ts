import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
