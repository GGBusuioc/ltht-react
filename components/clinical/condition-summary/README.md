# ConditionSummary

<!-- STORY -->

### Import

```js
import ConditionSummary from '@ltht-react/condition-summary'
```

### Usage

```jsx
<ConditionSummary title="Conditions" conditions={conditions} clickHandler={handleCLick} />
```

### Properties

| Prop           | Required | Default   | Type        | Description                                                   |
| :------------- | :------- | :-------- | :---------- | :------------------------------------------------------------ |
| `conditions`   | Yes      |           | Condition[] | Array of conditions to display                                |
| `clickHandler` | No       | undefined | Function    | Callback click handler containing the selected condition item |
