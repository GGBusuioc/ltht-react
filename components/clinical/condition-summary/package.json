{
  "name": "@ltht-react/condition-summary",
  "version": "0.0.3",
  "description": "ltht-react organisms ConditionSummary component.",
  "author": "LTHT",
  "homepage": "",
  "license": "MIT",
  "main": "lib/index.js",
  "typings": "lib/index.d.ts",
  "files": [
    "lib"
  ],
  "directories": {
    "lib": "lib"
  },
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "clean": "rimraf lib",
    "build": "yarn build:lib",
    "watch": "tsc -w",
    "build:lib": "tsc",
    "type:check": "tsc --noEmit",
    "lint": "prettylint 'src/*.{ts,tsx}'"
  },
  "dependencies": {
    "@emotion/core": "^10.0.27",
    "@emotion/styled": "^10.0.27",
    "@ltht-react/card": "^0.1.2",
    "@ltht-react/list": "^0.1.3",
    "@ltht-react/styles": "^0.1.0",
    "@ltht-react/types": "^0.0.9",
    "@ltht-react/utils": "^0.1.2",
    "emotion": "^10.0.27",
    "react": "^16.12.0"
  }
}
