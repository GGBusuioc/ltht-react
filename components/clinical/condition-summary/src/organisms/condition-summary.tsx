import React, { HTMLAttributes } from 'react'
import styled from '@emotion/styled'

import { Condition } from '@ltht-react/types'
import { DateSummary } from '@ltht-react/summary'

import Category from '../atoms/condition-category'
import Status from '../atoms/condition-status'
import Title from '../atoms/condition-title'
import Redacted from '../molecules/condition-redacted'

const StyledSummary = styled.div`
  display: flex;
  justify-content: center;
`
const StyledDescription = styled.div`
  flex-grow: 1;
`
const StyledDate = styled.div`
  text-align: right;
`

const ConditionSummary: React.FC<Props> = ({ condition, ...rest }) => {
  if (condition.metadata.isRedacted) {
    return (
      <StyledSummary {...rest}>
        <Redacted condition={condition} />
      </StyledSummary>
    )
  }

  return (
    <StyledSummary {...rest}>
      <StyledDescription>
        <Title condition={condition} />
        <Category condition={condition} />
      </StyledDescription>
      <StyledDate>
        <DateSummary datetime={condition?.assertedDate} />
        <Status condition={condition} />
      </StyledDate>
    </StyledSummary>
  )
}

interface Props extends HTMLAttributes<HTMLDivElement> {
  condition: Condition
}

export default ConditionSummary
