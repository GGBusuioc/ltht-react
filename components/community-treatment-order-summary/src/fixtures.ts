import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
