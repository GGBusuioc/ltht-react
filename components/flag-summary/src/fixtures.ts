import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
