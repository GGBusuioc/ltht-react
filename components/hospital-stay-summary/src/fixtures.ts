import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
