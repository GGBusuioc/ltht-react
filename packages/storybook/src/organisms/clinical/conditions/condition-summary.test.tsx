import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import ConditionSummary from '@ltht-react/condition-summary'
import { ConditionOne, ConditionTwo } from './condition-summary.fixtures'

describe('ConditionSummary', () => {
  const conditions = [ConditionOne, ConditionTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ConditionSummary title="Conditions" conditions={conditions} />, div)
  })
  it('matches snapshot', () => {
    expect(mount(<ConditionSummary title="Conditions" conditions={conditions} />)).toMatchSnapshot('wrapper mount')
  })
})
