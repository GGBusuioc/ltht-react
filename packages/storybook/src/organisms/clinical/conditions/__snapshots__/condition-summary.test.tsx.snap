// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConditionSummary matches snapshot: wrapper mount 1`] = `
.emotion-14 {
  color: #231f20;
  font-family: 'SF Pro Text','SF Pro Icons','Helvetica Neue',Helvetica,Arial,sans-serif;
  font-size: 0.8rem;
  background: #FFFFFF;
  color: #231f20;
  margin-bottom: 0.5rem;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(102,102,102,0.1), 0px 1px 1px 0px rgba(102,102,102,0.15), 0px 1px 3px 0px rgba(102,102,102,.6);
  margin: 0 5px 10px 0;
  -webkit-font-smoothing: antialiased;
}

.emotion-14 *,
.emotion-14 ::before,
.emotion-14 ::after {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

.emotion-14 * {
  margin: 0;
  padding: 0;
}

.emotion-14 h3 {
  font-size: 1rem;
}

@media only screen and (min-width:768px) {
  .emotion-14 {
    font-size: 0.9rem;
  }

  .emotion-14 h3 {
    font-size: 1.1rem;
  }
}

@media only screen and (min-width:1024px) {
  .emotion-14 {
    font-size: 1rem;
  }

  .emotion-14 h3 {
    font-size: 1.2rem;
  }
}

@media only screen and (min-width:1200px) {
  .emotion-14 {
    font-size: 1.1rem;
  }

  .emotion-14 h3 {
    font-size: 1.3rem;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0.5rem;
}

.emotion-2:hover {
  cursor: pointer;
}

.emotion-0 {
  color: #768692;
}

.emotion-13 {
  list-style: none;
  padding-left: 0.5rem;
}

.emotion-13 li {
  border-top: 1px solid #b0b0b0;
  padding: 0.5rem 0.5rem 0.5rem 0;
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-5 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-3 {
  color: #231f20;
  text-align: left;
}

.emotion-4 {
  color: #768692;
  text-align: left;
}

.emotion-6 {
  color: #231f20;
  text-align: right;
}

<ConditionSummary
  conditions={
    Array [
      Object {
        "assertedDate": Object {
          "kind": "DATE",
          "value": "2020-02-01T00:00:00+00:00",
        },
        "clinicalStatus": "ACTIVE",
        "code": Object {
          "coding": Array [
            Object {
              "code": "60123008",
              "display": "Delusional disorder, mixed type",
              "system": "HEALTHPLUG",
            },
          ],
          "text": "Delusional disorder, mixed type",
        },
        "id": "017ca927-67e0-e581-a2c7-115087226fg2|condition",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "requestedWhen": "",
        },
        "subject": Object {
          "display": "Mt Peter Humble",
          "typeName": "test",
        },
      },
      Object {
        "assertedDate": Object {
          "kind": "DATE",
          "value": "2020-01-01T00:00:00+00:00",
        },
        "clinicalStatus": "REMISSION",
        "code": Object {
          "coding": Array [
            Object {
              "code": "44054006",
              "display": "Diabetes mellitus type 2",
              "system": "HEALTHPLUG",
            },
          ],
          "text": "Diabetes mellitus type 2",
        },
        "id": "345ha927-34e0-e911-a2c7-005087226er3|condition",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "requestedWhen": "",
        },
        "subject": Object {
          "display": "Mr Joe BLoggs",
          "typeName": "test",
        },
      },
    ]
  }
  title="Conditions"
>
  <Card
    noData={false}
  >
    <CardProvider
      collapsible={true}
      noData={false}
    >
      <div
        className="emotion-14"
      >
        <CardHeader>
          <div
            className="emotion-2"
            onClick={[Function]}
            title="Click to collapse"
          >
            <div>
              <h3>
                Conditions
              </h3>
            </div>
            <div>
              <ChevronCircleIcon
                direction="down"
                size="medium"
              >
                <FontAwesomeIcon
                  border={false}
                  className="emotion-0"
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        512,
                        512,
                        Array [],
                        "f13a",
                        "M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z",
                      ],
                      "iconName": "chevron-circle-down",
                      "prefix": "fas",
                    }
                  }
                  inverse={false}
                  listItem={false}
                  mask={null}
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size="sm"
                  spin={false}
                  swapOpacity={false}
                  symbol={false}
                  title=""
                  transform={null}
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-chevron-circle-down fa-w-16 fa-sm emotion-0"
                    data-icon="chevron-circle-down"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                </FontAwesomeIcon>
              </ChevronCircleIcon>
            </div>
          </div>
        </CardHeader>
        <CardList
          clickable={false}
        >
          <ul
            className="emotion-13"
          >
            <ListItem
              key="017ca927-67e0-e581-a2c7-115087226fg2|condition"
            >
              <li>
                <ConditionSummaryItem
                  condition={
                    Object {
                      "assertedDate": Object {
                        "kind": "DATE",
                        "value": "2020-02-01T00:00:00+00:00",
                      },
                      "clinicalStatus": "ACTIVE",
                      "code": Object {
                        "coding": Array [
                          Object {
                            "code": "60123008",
                            "display": "Delusional disorder, mixed type",
                            "system": "HEALTHPLUG",
                          },
                        ],
                        "text": "Delusional disorder, mixed type",
                      },
                      "id": "017ca927-67e0-e581-a2c7-115087226fg2|condition",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                        "requestedWhen": "",
                      },
                      "subject": Object {
                        "display": "Mt Peter Humble",
                        "typeName": "test",
                      },
                    }
                  }
                >
                  <div
                    className="emotion-7"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <ConditionTitle
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-02-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "ACTIVE",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "60123008",
                                  "display": "Delusional disorder, mixed type",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Delusional disorder, mixed type",
                            },
                            "id": "017ca927-67e0-e581-a2c7-115087226fg2|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mt Peter Humble",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          Delusional disorder, mixed type
                        </div>
                      </ConditionTitle>
                      <ConditionStatus
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-02-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "ACTIVE",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "60123008",
                                  "display": "Delusional disorder, mixed type",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Delusional disorder, mixed type",
                            },
                            "id": "017ca927-67e0-e581-a2c7-115087226fg2|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mt Peter Humble",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          Active
                        </div>
                      </ConditionStatus>
                    </div>
                    <div>
                      <ConditionDate
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-02-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "ACTIVE",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "60123008",
                                  "display": "Delusional disorder, mixed type",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Delusional disorder, mixed type",
                            },
                            "id": "017ca927-67e0-e581-a2c7-115087226fg2|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mt Peter Humble",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          01-Feb-2020
                        </div>
                      </ConditionDate>
                    </div>
                  </div>
                </ConditionSummaryItem>
              </li>
            </ListItem>
            <ListItem
              key="345ha927-34e0-e911-a2c7-005087226er3|condition"
            >
              <li>
                <ConditionSummaryItem
                  condition={
                    Object {
                      "assertedDate": Object {
                        "kind": "DATE",
                        "value": "2020-01-01T00:00:00+00:00",
                      },
                      "clinicalStatus": "REMISSION",
                      "code": Object {
                        "coding": Array [
                          Object {
                            "code": "44054006",
                            "display": "Diabetes mellitus type 2",
                            "system": "HEALTHPLUG",
                          },
                        ],
                        "text": "Diabetes mellitus type 2",
                      },
                      "id": "345ha927-34e0-e911-a2c7-005087226er3|condition",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                        "requestedWhen": "",
                      },
                      "subject": Object {
                        "display": "Mr Joe BLoggs",
                        "typeName": "test",
                      },
                    }
                  }
                >
                  <div
                    className="emotion-7"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <ConditionTitle
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-01-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "REMISSION",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "44054006",
                                  "display": "Diabetes mellitus type 2",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Diabetes mellitus type 2",
                            },
                            "id": "345ha927-34e0-e911-a2c7-005087226er3|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mr Joe BLoggs",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          Diabetes mellitus type 2
                        </div>
                      </ConditionTitle>
                      <ConditionStatus
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-01-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "REMISSION",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "44054006",
                                  "display": "Diabetes mellitus type 2",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Diabetes mellitus type 2",
                            },
                            "id": "345ha927-34e0-e911-a2c7-005087226er3|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mr Joe BLoggs",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          Remission
                        </div>
                      </ConditionStatus>
                    </div>
                    <div>
                      <ConditionDate
                        condition={
                          Object {
                            "assertedDate": Object {
                              "kind": "DATE",
                              "value": "2020-01-01T00:00:00+00:00",
                            },
                            "clinicalStatus": "REMISSION",
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "44054006",
                                  "display": "Diabetes mellitus type 2",
                                  "system": "HEALTHPLUG",
                                },
                              ],
                              "text": "Diabetes mellitus type 2",
                            },
                            "id": "345ha927-34e0-e911-a2c7-005087226er3|condition",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "requestedWhen": "",
                            },
                            "subject": Object {
                              "display": "Mr Joe BLoggs",
                              "typeName": "test",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          01-Jan-2020
                        </div>
                      </ConditionDate>
                    </div>
                  </div>
                </ConditionSummaryItem>
              </li>
            </ListItem>
          </ul>
        </CardList>
      </div>
    </CardProvider>
  </Card>
</ConditionSummary>
`;
